proc-macros/tests/ui/correct/alias_doesnt_use_namespace.rs

```diff
@@ -1,12 +1,12 @@
-use jsonrpsee::{proc_macros::rpc, core::RpcResult};
+use jsonrpsee::{core::RpcResult, proc_macros::rpc};
 
 #[rpc(client, server, namespace = "myapi")]
 pub trait Rpc {
 	/// Alias doesn't use the namespace so not duplicated.
 	#[method(name = "getTemp", aliases = ["getTemp"])]
```

Comment:
I think we should clarify this comment a bit. This macro invocation generates two methods: `myapi_getTemp` and `getTemp` right?